package.json

```diff
@@ -207,7 +207,7 @@
     "balena-image-manager": "^7.0.3",
     "balena-preload": "^10.5.0",
     "balena-release": "^3.2.0",
-    "balena-sdk": "^15.48.0",
+    "balena-sdk": "^15.51.0",
```

Comment:
@nitishagar, Kyle's PRs have been merged and `balena-sdk` v15.51.1 is now available. Please select `balena-sdk` v15.51.1 here and change the commit `Change-type` to patch, then we can get this PR merged. Thanks! 🙏